ben.html

```diff
@@ -21,14 +21,66 @@ <h1>Bugaboos' Favorite Restaurants</h1>
 		
 		<article class="studentRestaurants">
 			<header class="studentHeader">
-				<h2>Ben's Restaurant Recommendations</h2>
+				<h2>Ben's Picks</h2>
 			</header>
 			<div class="restaurants">
 				<div class="restaurantContainer">
-					<h2>Restaurant</h2>
-					<div class="restaurantImage"></div>
-					<div class="restaurantInfo"></div>
-					<div class="restaurantLocations"></div>
+					<h2>Sushin</h2>
+					<div class="restaurantImage">
+						<img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.YtnW8AuvgJFq7hUSp2r6-QHaFj%26pid%3DApi&f=1" >
+					</div>
+					<div class="restaurantInfo">
+						<ul>
+							<li>Asian</li>
```

Comment:
I would add a Type: for the types of food for each restaurant, just because the other items listen have a "description:"